hugolib/page__tree.go

```diff
@@ -178,6 +178,15 @@ func (pt pageTree) Parent() page.Page {
 	return b.p
 }
 
+func (pt pageTree) Ancestors() (parents page.Pages) {
+	parent := pt.Parent()
+	for parent != nil {
+		parents = append(parents, parent)
```

Comment:
Nitpicking, but 

* `parents` should be named something else
* we only use named return values when it adds value (e.g. when we need to capture a panic and assign to err), so I would prefer `Ancestors() page.Pages`

Reply:
OK, I can do it. I need to merge this soonish, as I'm releasing today.

Reply:
Sorry, I got COVID-19 and I'm fine now